extensions/2.0/Khronos/KHR_materials_anisotropy/schema/glTF.KHR_materials_anisotropy.schema.json

```diff
@@ -7,11 +7,11 @@
     "properties": {
         "anisotropyStrength": {
             "type": "number",
-            "description": "The anisotropy strength. When anisotropyTexture is present, this value is multiplied by the blue channel.",
+            "description": "Anisotropic strength",
             "default": 0.0,
             "minimum": 0.0,
             "maximum": 1.0,
-            "gltf_detailedDescription": ""
+            "gltf_detailedDescription": "The anisotropy strength. When anisotropyTexture is present, this value is multiplied by the blue channel."
```

Comment:
Please either format `anisotropyTexture` with backticks or replace it with plain text, i.e., "anisotropy texture". Also consider changing "the blue channel" to "its blue channel" for more clarity.

Reply:
JSON schema descriptions aren't markdown, so backticks don't work here.  I can add a space or change the wording, but that will impact the README which bears a copy of this description as-is.

Reply:
OK, let's update the both the detailed description and the readme later. Backticks are used in the core schemas and that JSON is then translated into AsciiDoc property reference.